benches/search_benches.rs

```diff
@@ -0,0 +1,217 @@
+#![feature(test)]
+
+extern crate test;
+
+use cheminee::command_line::indexing::bulk_index::create_tantivy_doc;
+use cheminee::command_line::indexing::split_path;
+use cheminee::indexing::index_manager::IndexManager;
+use cheminee::indexing::{open_index, KNOWN_DESCRIPTORS};
+use cheminee::schema::LIBRARY;
+use cheminee::search::basic_search::basic_search;
+use cheminee::search::compound_processing::standardize_smiles;
+use cheminee::search::identity_search::identity_search;
+use cheminee::search::scaffold_search::{scaffold_search, PARSED_SCAFFOLDS};
+use cheminee::search::structure_search::structure_search;
+use cheminee::search::{aggregate_query_hits, prepare_query_structure, StructureSearchHit};
+use rayon::prelude::*;
+use serde_json::Value;
+use std::collections::{HashMap, HashSet};
+use std::ops::Deref;
+use tantivy::schema::Field;
+use test::Bencher;
```

Comment:
This include block is getting unwieldly, I'll do a future PR to reformat our project. Just a note to myself here...